unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs

```diff
@@ -84,35 +84,38 @@ public void Update_Trigger()
                                        InputProcessor.Modifier.FocusNotInInput);
                     break;
                 case DCLAction_Trigger.ToggleNavMap:
-                    if (allUIHidden) break;
+                    if (allUIHidden || CommonScriptableObjects.featureKeyTriggersBlocked) break;
```

Comment:
Let me suggest an alternative, less hacky approach:

Add a new `BoolVariable` field to the `ActionTrigger_Input` SO, so it gets enabled/disabled according to a custom `BoolVariable` object.

Just query that field value at the `for` start, like this:
```
if ( action.boolEnableObject != null && !action.boolEnableObject.Get() )
    continue;
```
With this in place, we can safely remove all the hardcoded conditions and make this feature data-driven.

Reply:
This a very elegant solution for this! Thanks for the suggestion :) I have pushed the change, please take a look!